wp-admin/js/widgets/media-widgets.js

```diff
@@ -614,20 +614,20 @@ wp.mediaWidgets = ( function( $ ) {
 				} else {
 					castedAttrs[ name ] = value;
 				}
-			} );
+			});
 
 			return Backbone.Model.prototype.set.call( this, castedAttrs, opts );
 		}
-	} );
+	});
 
 	/**
 	 * Collection of all widget model instances.
 	 *
 	 * @type {Backbone.Collection}
 	 */
-	component.modelCollection = new ( Backbone.Collection.extend( {
+	component.modelCollection = new ( Backbone.Collection.extend({
 		model: component.MediaWidgetModel
-	} ) )();
+	}) )();
```

Comment:
Maybe, maybe not. The convention is to only collapse whitespace following the brace. There's no convention for this specific case other than the blanket:

> Use spaces liberally throughout your code. “When in doubt, space it out.”
>
> These rules encourage liberal spacing for improved developer readability.

So I think unless the standards explicitly say there should be no space, then there _should_ be a space.